.github/workflows/linux.yml

```diff
@@ -133,20 +133,28 @@ jobs:
         with:
           submodules: recursive
 
-      - name: Update version
-        shell: bash
-        run: |
-          sed -i.bak "s/[^_]PROJECT_VERSION *[^)]\+/&-g${GITHUB_SHA::7}/" CMakeLists.txt
-
       - name: Setup variables
+        shell: bash
         id: vars
         run: |
-          version=$(grep -m 1 '[^_]PROJECT_VERSION' CMakeLists.txt | sed 's/.*[^_]PROJECT_VERSION *//' | sed 's/).*//')
+          set -x
```

Comment:
Is this still necessary?

Reply:
Ah, yes. I forgot to mention that. I explicitly set this to make scripting errors stand out more. It's a bit verbose, I guess, but at the same time our scripts are rather small.